packages/shared/src/hooks/usePostModalNavigation.ts

```diff
@@ -106,8 +114,18 @@ export const usePostModalNavigation = (
     };
   }, [items]);
 
+  const getPostPosition = () => {
+    const isPost = (item: FeedItem) => item.type === 'post';
```

Comment:
Maybe we should enum this too?
We already have `SourceType` so maybe `PostType`?

Although thinking about it, does it matter if it's post only, the sharedpost should also have the same behaviour right?

Reply:
Shared post is still a post. This is not checking the post.type. This is checking feeditem.type

Reply:
Got it 🙏